src/Algebra/Bundles.agda

```diff
@@ -883,6 +883,113 @@ record CommutativeRing c ℓ : Set (suc (c ⊔ ℓ)) where
     )
 
 ------------------------------------------------------------------------
+-- Bundles with 3 binary operations
+------------------------------------------------------------------------
+
+record RawQuasiGroup c ℓ : Set (suc (c ⊔ ℓ)) where
+  field
+    Carrier : Set c
+    _≈_     : Rel Carrier ℓ
+    _∙_     : Op₂ Carrier
+    _\\_    : Op₂ Carrier
```

Comment:
Out of curiosity is there a reason for having double slashes instead of single slashes?

Reply:
To use single slash we get this "In the name _\_, the part \ is not valid because it is used for lambda-abstraction".

Reply:
Of course that makes sense, thanks!